code/src/components/TodoItem.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import moment from 'moment';
+import styled from 'styled-components/macro';
+
+import { todos } from '../reducers/todos';
+import { TrashBin } from '../lib/Image';
+import { Span } from '../lib/Text';
+import CustomCheckbox from '../lib/CustomCheckbox';
+
+const TodoItem = ({ item }) => {
+  const [active, setActive] = useState('');
+  const dispatch = useDispatch();
+
+  const handleRemoveTodo = () => {
+    dispatch(
+      todos.actions.removeTodo({
+        id: item.id,
+      })
+    );
+  };
+  
+  const toggleComplete = () => {
+    dispatch(todos.actions.setComplete(item.id));
+  };
+
+  const toggleAccordion = () => {
+    setActive(active === '' ? 'active' : '');
+  };
+
+  return (
```

Comment:
FYI: Something in this return is creating a semicolon
![Skärmavbild 2020-12-16 kl  10 56 12](https://user-images.githubusercontent.com/51529201/102333149-5097db00-3f8d-11eb-855f-ee26b369e2f0.png)